README

```diff
@@ -66,6 +66,17 @@ or application code.
 (n.b. these examples still need extending to show how to use Byteman
 for fault injection based testing).
 
+Compiling Byteman
+-----------------
+
+To compile source code use Maven command `mvn clean install`.
+Byteman will be compiled, the command generates documentation
+and distribution artifacts can be found at `download/target`.
+
+To speed-up compilation you can skip tests and generating
+the documentation with
+`mvn clean install -DskipITs -DskipTests -DnoDoc`.
```

Comment:
This is a very nice idea. It might be better to use -DskipDocs instead of -DnoDocs.

Reply:
sure, changed to `-DskipDocs`